.changelog/4696.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: YmlSplitter fix.
```

Comment:
Fixed an issue where YmlSplitter...(complete.)

Reply:
Only fixing the pr number, the changelog description was already adapted in the released changelog.